ckanext/tracking/cli/tracking.py

```diff
@@ -142,10 +144,12 @@ def update_tracking(summary_date: datetime.datetime):
     Update the tracking_summary table with data from tracking_raw
     '''
     package_url = "/dataset/"
+    root_path = config.get('ckan.root_path')
+    root_path = re.sub('/{{LANG}}', '', root_path)
```

Comment:
hey @Nisha129312 
can we check for the `ckan.root_path` if it was set and in that case to do all replacing ? If we dont have `root_path` set we are doing unnecessary replace in later in the code

Reply:
@tino097 I have done the changes as per your comment. Can you please now review it.

Reply:
Maybe you could try something like this 
```
rp = config.get('ckan.root_path', '')
root_path = re.sub('/{{LANG}}', '', rp) if rp else ''
url = (func.replace(tr.url, root_path, '').label("tracking_url") if root_path else tr.url)
```
and use `url` in the query